docs/intro/tutorial.rst

```diff
@@ -83,8 +83,8 @@ sites, we define fields for each of these three attributes. To do that, we edit
         link = scrapy.Field()
         desc = scrapy.Field()
 
-This may seem complicated at first, but defining the item allows you to use other handy
-components of Scrapy that need to know what does your item look like.
+This may seem complicated at first, but defining an item class allows you to use other handy
+components and helpers within Scrapy.
```

Comment:
Hello @rhoekman -- thanks for checking our docs!

I'm not sure if I understand what exactly this is fixing, could you explain? (non-native English speaker here, sorry if I'm missing something)

The idea here was to state that some Scrapy components need to know what your item looks like (namely, the item schema), I feel this note is being lost in this new version.